app/uk/gov/hmrc/gform/summarypdf/PdfGeneratorService.scala

```diff
@@ -38,6 +40,9 @@ class PdfGeneratorService(
     StreamConverters.asOutputStream().mapMaterializedValue { os =>
       Future {
         val builder = new PdfRendererBuilder()
+        builder.usePdfUaAccessbility(true)
+        builder.usePdfAConformance(PdfRendererBuilder.PdfAConformance.PDFA_3_U)
+        builder.useFont(() => environment.classLoader.getResourceAsStream("arial.ttf"), "Arial")
```

Comment:
Do we have `arial.ttf` on classpath in production? I may be wrong, but I thought that they are only shipped with Oracle java, not with openjdk.

Reply:
Sorry, I messed this.